recipes/hyperscan/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conans import ConanFile, CMake, tools
+
+import os
+
+class HyperscanConan(ConanFile):
+    name = "hyperscan"
+    license = "BSD-2-Clause"
```

Comment:
https://github.com/intel/hyperscan/blob/64a995bf445d86b74eb0f375624ffc85682eadfe/LICENSE#L13

https://spdx.org/licenses/BSD-3-Clause.html

```suggestion
        license = "BSD-3-Clause"
```